TLM/TLM/Patch/_VehicleAI/_BankVanAI/StartPathFindPatch.cs

```diff
@@ -0,0 +1,29 @@
+namespace TrafficManager.Patch._VehicleAI._BankVanAI {
+    using System.Collections.Generic;
+    using System.Reflection;
+    using System.Reflection.Emit;
+    using HarmonyLib;
+    using JetBrains.Annotations;
+    using TrafficManager.API.Traffic.Enums;
+    using TrafficManager.Patch._PathManager;
+
+    [HarmonyPatch]
+    public class StartPathFindPatch {
+
+        [UsedImplicitly]
+        public static MethodBase TargetMethod() => StartPathFindCommons.TargetMethod<BankVanAI>();
+
+        [UsedImplicitly]
+        public static void Prefix(ushort vehicleID, ref Vehicle vehicleData) {
+            CreatePathPatch.ExtPathType = ExtPathType.None;
+            CreatePathPatch.ExtVehicleType = ExtVehicleType.Service;
+            CreatePathPatch.VehicleID = vehicleID;
+        }
+
+        [UsedImplicitly]
+        [HarmonyPriority(Priority.Low)] // so that if this code is redundant, it would result in warning log.
```

Comment:
>         [HarmonyPriority(Priority.Low)] // so that if this code is redundant, it would result in warning log.

priority is for order not for throwing exception.

Reply:
How is that related with throwing exception?? Please read that comment again... 
Anyways, it's your code, just copy pasted from PostVanAI, which has been squashed from harmony migration branch you've created long time ago.